### PR TITLE
JBPM-5991 - Add shortcut for predefined values to WID editor

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@hasys can you please review? thanks!

File: drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-backend/src/main/java/org/drools/workbench/screens/workitems/backend/server/WorkbenchConfigurationHelper.java [hunk 1]
```diff
@@ -61,11 +61,22 @@ public ConfigGroup getWorkItemElementDefinitions() {
...
         group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_RESULT,
-                                                                 "\"Result|\" : new ObjectDataType()" ) );
+                                                                 "\"Result\" : new ObjectDataType()" ) );
         group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_DISPLAY_NAME,
-                                                                 "\"displayName\" : \"My Task|\"" ) );
+                                                                 "\"displayName\" : \"My Task\"" ) );
+
+        group.addConfigItem( configurationFactory.newConfigItem( WorkItemsEditorService.WORK_ITEMS_EDITOR_SETTINGS_PARAMETER_VALUES,
+                "\"parameterValues\" : [\n" +
+                        "   \"MyParam\" : \"A,B,C\",   \n" +
```
Comment (hasys) [inline drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-backend/src/main/java/org/drools/workbench/screens/workitems/backend/server/WorkbenchConfigurationHelper.java]: Definition shortcut have parameters names:
```javascript
    "parameters" : [ 
        "MyFirstParam" : new StringDataType(), 
        "MySecondParam" : new StringDataType(), 
        "MyThirdParam" : new ObjectDataType() 
    ]
```
I guess we can use "MyFirstParam" as default name here

File: drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-backend/src/main/java/org/drools/workbench/screens/workitems/backend/server/WorkbenchConfigurationHelper.java [hunk 2]
```diff
@@ -61,11 +61,22 @@ public ConfigGroup getWorkItemElementDefinitions() {
...
+                        "   \"MySecondParam\" : \"X,Y,Z\"\n" +
+                        "]," ) );
```
Comment (hasys) [inline drools-wb-screens/drools-wb-workitems-editor/drools-wb-workitems-editor-backend/src/main/java/org/drools/workbench/screens/workitems/backend/server/WorkbenchConfigurationHelper.java]: parameterValues is the only shortcut with ',' on the end

**Comment by tsurdilo** [comment]
@hasys amended

**Comment by tsurdilo** [comment]
@hasys yeah i agree - as we talked about i think
"icon" : ""
is fine and we need jira for designer to fix that there. empty value for icon or no icon attribute should 
default to defaultserviceicon there

**Comment by tsurdilo** [comment]
@manstis the validWorkItemDefinition.wid file in this PR is used by a test. there was imo no need to add extra test methods but just make sure that existing test with the addition of the two added attributes run fine.

**Comment by manstis** [comment]
@tsurdilo I see changes in ```WorkDefinitionsParser``` and ```WorkbenchConfigurationHelper``` that are not tests; nor can I see the changes covered by tests. If however there is some other magic going on where code is tested automatically, please point me in the right direction :-)

**Comment by tsurdilo** [comment]
@manstis amended. thanks!

**Pull request merged**